tests/core/cgo/BUILD.bazel

```diff
@@ -331,6 +332,38 @@ go_binary(
     cgo = True,
 )
 
+MANY_EXT_LDFLAGS = [
+    flag
+    for i in range(5)
+    for flag in [
+        "-extldflags",
+        "-Wl,-rpath,$$ORIGIN/path/to/lib/{}".format(i),
+    ]
+]
+
+go_binary(
```

Comment:
Is there a way to write this so the test is not Linux-specific? The current test uses Bash and assumes ELF.

Ideally, we'd have a `go_test` that includes a `.cc` file (and so is cgo). The `-extldflags` would be long enough to exceed the 32k limit but also observable in some way to the Go part of the test.

Not sure if `-rpath` will work for this. Is there something equivalent to Go's `-X` flag we could use?

If this isn't possible, I'd still be happy with just a `go_binary` with long enough flags to exceed the limit: we'd just be checking that it builds successfully.

Reply:
Trying to understand the ask here.

Are you suggesting that I drop both these tests?
- `sh_test` since it's Linux-specific.
- `cgo_ldflags_test` `go_test` since the linker values are passed via cgo flags.

And just keep `binary_with_rpath` `go_binary`?

Reply:
Done!